lib/relations/RelationProperty.js

```diff
@@ -182,7 +182,9 @@ function createPaths(refs, modelClassResolver) {
     const modelClass = modelClassResolver(ref.tableName);
 
     if (!modelClass) {
-      throw new ModelNotFoundError();
+      const error = new ModelNotFoundError();
+      error.tableName = ref.tableName;
```

Comment:
This property should be added to the class and passed to the constructor:

```js
class ModelNotFoundError extends Error {
  constructor(tableName) {
    super();
    this.tableName = tableName;
  }
}
```

Reply:
May I quickly ask why/the reason for this?